src/seedu/addressbook/data/person/address/Street.java

```diff
@@ -0,0 +1,39 @@
+package seedu.addressbook.data.person.address;
+
+public class Street {
+
+	private String streetAddress;
+
+	public Street(String address){
+		streetAddress = address;
+	}
```

Comment:
Better to change your parameter to `street`, since it doesn't make sense to put address into street which two different things. So you can write like this, `this.street = street`